release-repo-scripts/merge_forward_pull_request.bash

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+# Copyright (C) 2022 Open Source Robotics Foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# The script will merge forward and open a pull request.
+#
+# Requires the 'gh' CLI to be installed.
+#
+# Usage:
+# $ ./merge_forward_pull_request.bash <from_branch> <to_branch>
+#
+# For example, to merge `ign-rendering6` forward to `main`:
+#
+# ./merge_forward_pull_request.bash ign-rendering6 main
+
+FROM_BRANCH=${1}
+TO_BRANCH=${2}
+
+set -e
+
+ORIGIN_URL=$(git remote get-url origin)
+ORIGIN_ORG_REPO=$(echo ${ORIGIN_URL} | sed -e 's@.*github\.com.@@')
+
+TITLE="Merge ${FROM_BRANCH} -> ${TO_BRANCH}"
```

Comment:
Can we use ➡️ here? I find it easier to spot when looking through the PR list.

```suggestion
TITLE="Merge ${FROM_BRANCH} ➡️ ${TO_BRANCH}"
```

Reply:
https://github.com/gazebo-tooling/release-tools/pull/720/commits/97845bac0fe1b27d2fd26a5e171cd92c912f41a9

Reply:
That's not in the title though 😢

Reply:
sorry, it should be in the title as of https://github.com/gazebo-tooling/release-tools/pull/720/commits/b517bbdd8db92541f36a69c6cfc73de74934f254